setup.py

```diff
@@ -70,7 +70,7 @@ def build(cls):
         try:
             import sass
         except ModuleNotFoundError:
-            pip.main(['install', 'libsass'])
+            check_call([sys.executable, '-m', 'pip', 'install', 'libsass'])
```

Comment:
Now, I understood. Thanks @diraol for the explanation.

Please lets add libass on requirements-build.txt and install from there. Dependencies on source code is not optimal.

Thanks.